express-static/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const app = express();
+
+const path = require('path');
+
+// eslint-disable-next-line no-unused-vars
+const absolutePath = path.join('_dirname', 'public');
+
+// eslint-disable-next-line no-unused-vars
+const middleware = app.use(express.static('public'));
```

Comment:
These ESLint directives shouldn't be necessary. We want to pass the `absolutePath` to `express.static()` instead of just `'public'` and you can ignore the return value of `app.use()`.

Reply:
I see! Will fix this